test/h.js

```diff
@@ -163,3 +163,10 @@ test.failing('don\'t merge children of components', t => {
 
 	t.deepEqual(node.props.children, ['x', 'y']);
 });
+
+test('warns on invalid type', t => {
+	t.throws(() => {
```

Comment:
=> `t.throws(() => h(), 'Expected component to be a function, got undefined. You may have forgotten to export a component.');`